docs/html/topics/configuration.md

```diff
@@ -19,13 +19,14 @@ and how they are related to pip's various command line options.
 
 ## Configuration Files
 
-Configuration files can change the default values for command line option.
-They are written using a standard INI style configuration files.
+Configuration files can change the default values for command line options.
+They are written using standard INI style configuration files.
 
-pip has 3 "levels" of configuration files:
+pip has 4 "levels" of configuration files:
 
-- `global`: system-wide configuration file, shared across users.
-- `user`: per-user configuration file.
+- `global`: system-wide configuration file, shared across all users.
+- `user`: per-user configuration file, shared across all environments.
+- `base` : per-base environment configuration file, shared across all virtualenvs with the same base.
```

Comment:
It would be a good idea to add a note somewhere that this is added in $VERSION.

Reply:
I wrote ``v23``, but don't know if that is correct or not.

Reply:
I doubt this would be ready in time for 22.3[^1] (the last release this year) so that seems reasonable. We can update the exact version number when it's ready to merge. 

[^1]: In particular, with my 22.3 RM hat on, please don't rush to get it ready for 22.3, it seems complex enough that I'd prefer it to wait a while longer.